components/skin/ui/src/main/resources/PhenoTips/ContentTopMenu.xml

```diff
@@ -859,6 +859,8 @@
 ##
 #if ($doc.getObject('PhenoTips.PatientClass'))
 $xwiki.jsx.use('PhenoTips.ContentTopMenu')##
+$xwiki.jsx.use('PhenoTips.JSONImportDialog')##
```

Comment:
These should only be used when the import is actually exposed, so move them inside the `#if` below.